### PR TITLE
Feature/102 three search endpoints

### PR DESCRIPTION
**Pull request by YoYuUm**
It's still quite messy, but at least you can take a look. @lraivio

### CODE REVIEW

File: app/api/SearchAPI.js [hunk 1]
```diff
@@ -25,5 +60,22 @@ export default {
           SearchServerActionCreators.handleFilterError(response.text);
         }
       });
+  },
+
+  searchLoadMore(urls) {
+    console.log(urls);
```
Comment (lraivio) [inline app/api/SearchAPI.js]: This could be removed once everything is ready.

File: app/pages/SearchPage.js [hunk 2]
```diff
@@ -90,12 +100,14 @@ class SearchPage extends React.Component {
...
   render() {
+    console.log(this.state.principalList);
+    console.log(this.state.schoolBuildingList);
     return (
       <DocumentTitle title='Etsi koulua - Koulurekisteri'>
         <div className='search-page'>
           <header className='container'>
             <h1 className='search-title' >
-              <img alt='Helsingin vaakuna' src={require('../images/Helsinki.vaakuna.svg')} />
+              <img alt='Helsinkingin vaakuna' src={require('../images/Helsinki.vaakuna.svg')} />
```
Comment (lraivio) [inline app/pages/SearchPage.js]: This should be "Helsingin vaakuna"

Comment (YoYuUm) [inline app/pages/SearchPage.js]: I don't understand how did this happen.. hehe

**Comment by lraivio** [comment]
These commits were added in #119.

**Pull request closed without merging**